Modules/rat_command.py

```diff
@@ -63,6 +64,9 @@ class NameCollisionException(CommandException):
 # Pydle bot instance.
 bot: BasicClient = None
 
+# facts manager instance
+facts_m: FactsManager = FactsManager()
```

Comment:
please rename this to `facts_manager`, `facts_m` doesn't exactly describe what this field represents

Reply:
done